patches/common/chromium/.patches.yaml

```diff
@@ -273,6 +273,10 @@ patches:
   owners: deepak1556
   file: disable-recursive-surface-sync.patch
   description: null
+-
+  owners: brenca
+  file: disable-redraw-lock.patch
+  description: null
```

Comment:
It would be better to have an explanation why it is needed.
For instance a link to an issue it fixes, or a feature it implements.

Reply:
On it!